packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItem.tsx

```diff
@@ -48,6 +48,10 @@ const useTreeItemStyle = makeStyles((theme) => ({
       backgroundColor: 'var(--blue-lagoon-lighten-90)',
     },
   },
+  nonSelectable: {
+    fontStyle: 'italic',
+    opacity: 0.6,
+  },
```

Comment:
Would it be interesting to apply the cursor pointer only to selectable elements rather than to all labels, as is currently the case?

Reply:
Not sure what you mean.

Reply:
With the style `imageAndLabel`, `cursor: 'pointer'` is applied on all the item labels even those that are not selectable.
Wouldn't it be  more logic to add this style only for the selectabled one ?

Reply:
There's [a separate PR](https://github.com/eclipse-sirius/sirius-components/pull/2131) for this. I need it here in the commit stack, but its not directly in the scope of the "[2116] Add a modal to select a new value for single-valued references" work.